lib/csv2avro/converter.rb

```diff
@@ -1,14 +1,20 @@
 require 'csv2avro/schema'
 require 'csv2avro/avro_writer'
 require 'csv'
+require 'logr'
 
 class CSV2Avro
   class Converter
-    def initialize(reader, writer, bad_rows_writer, error_writer, options, schema: schema)
+
+    def self.logger
```

Comment:
When you refactor you might want to create an instance method to access the logger.

```ruby
def logger
  self.class.logger
end
```

So you don't have write `CSV2Avro::Converter.logger` all the time.

It's fine now, just mentioning it for future reference.